.github/workflows/visual-regression-test.yml

```
@@ -41,25 +49,12 @@ jobs:
           --outDir packages/spindle-ui/.reg-suit/actual_images/
           --serverCmd 'npx serve -p 5000 packages/spindle-ui/public'
           http://localhost:5000
-      - name: checkout branch when push
-        if: github.event_name == 'push'
-        run: |
-          git checkout "${GITHUB_REF#refs/heads/}"
-      - name: checkout branch when create Pull-request
-        if: github.event_name == 'pull_request'
-        env:
-          REF: ${{ github.head_ref }}
-        run: |
-          git checkout "${REF}"
-      - run: git symbolic-ref --short HEAD
-      # NOTE: Don't execute if sent Pull-request from openameba/spindle
-      - if: |
-          !(github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == 'openameba/spindle')
+      - run: git checkout ${{ github.head_ref || github.ref_name }}
+      - run: |
+          printf -- '%s' "$GOOGLE_APPLICATION_CREDENTIALS_JSON" | base64 --decode > "${PWD}/storage-key.json"
```

めちゃ細かいんですが
```suggestion
          printf -- '%s' '$GOOGLE_APPLICATION_CREDENTIALS_JSON' | base64 --decode > '${PWD}/storage-key.json'
```
シングルクオートの方がよかったりします？

---

Bashなのでシングルクォートにすると変数が展開されなくなっちゃいまして。この箇所では両方とも変数を使ってるのでダブルクォートで囲わないとですね。